### ISSUE TITLE
404 pages have a broken image tag

### ISSUE THREAD

**Issue by vaidehijoshi** [emberjs/guides#1952]
Hi there!

While perusing through the Ember guides, I stumbled upon the 404 page via [this](https://guides.emberjs.com/v2.13.0/controllers/dependencies-between-controllers/) url. I noticed that the image tag is broken:

<img width="837" alt="screen shot 2017-05-22 at 6 26 17 pm" src="https://cloud.githubusercontent.com/assets/6921610/26334829/a7aebe60-3f1c-11e7-906b-440ab93b42e9.png">

I'd be happy to submit a PR for this, if you could just point me to the correct url. It would be my first Ember contribution, seems like a nice easy fix to start with !😸

**Comment by locks** [comment]
This should be fixed by https://github.com/emberjs/guides/pull/1940/files 🤔

**Comment by vaidehijoshi** [comment]
@locks Weird. I took a closer look at this, seems like the URL that I hit when I encountered this doesn't pick up on the changes made in that PR. 

The image `src` is pointing to "http://emberjs.com/images/tomsters/fishy.png", while it ought to be pointing to "https://guides.emberjs.com/v2.13.0/images/fishy.png".

What I'm confused about is why, when I run it locally, everything works just fine. 😳
